### ISSUE TITLE
TemplateDoesNotExist at / bootstrap4/messages.html

### ISSUE THREAD

**Issue by egorsmkv** [zostera/django-bootstrap4#706]
I am using the following code in https://github.com/egorsmkv/simple-django-login-and-register/blob/master/source/content/templates/layouts/default/page.html#L77

```
    {% bootstrap_messages %}
```

but getting this error now after updating the library to **24.2**:

```
TemplateDoesNotExist at /
bootstrap4/messages.html
```

**Comment by egorsmkv** [comment]
When I downgrade the library to **24.1**, then it works as expected

**Comment by dyve** [comment]
It looks like the templates are no longer included in the package. Thank you for reporting, n=investigating. Please pin 24.1 as a workaround.

**Comment by dyve** [comment]
Dropping `setuptools_scm` caused this. We don't need this for version tracking, but it seems it does make it easier to include all necessary data. I'll try a fix by bringing `setuptools_scm` back.

**Comment by dyve** [comment]
Fixed in 24.3. Apologies.

**Comment by egorsmkv** [comment]
Thank you @dyve !
